tree/master/cloud/src/solution/Deployment/Scripts/Deploy-AzureResourceGroup.ps1

```diff
@@ -25,7 +30,7 @@ $vaults_KeyVault_name = $WebAppName+"KeyVault"
 $storageAccount_name = $WebAppName
 $serverfarms_WebPlan_name = $WebAppName+"WebPlan"
 $ADApp_Name = $WebAppName+"ADApp"
-
+$global:thumbPrint = ""
```

Comment:
global variable for thumbprint which will be used in matter web app settings deployment